app/data/models/Deck.py

```diff
@@ -0,0 +1,29 @@
+from app.data.repositories.UsersRepository import user
+from app.data.repositories import CardsRepository
+
+class Deck:
+
+
+    def __init__(self, deck_id, user_id, name):
+        
+        self.deck_id = deck_id
+        self.user_id = user_id
+        self.name = name
+
+
+    @classmethod
+    def new_deck(cls, user_id, name):
+        return cls(None, user_id, name)
+
+
+    @classmethod
+    def deck(cls, args):
```

Comment:
whats that?

Reply:
like with other models. It gives me an object of deck created by passing arguments from the database.